src/test/java/com/google/devtools/build/lib/remote/RemoteSpawnRunnerTest.java

```diff
@@ -145,7 +146,7 @@
 
   // The action key of the Spawn returned by newSimpleSpawn().
   private final String simpleActionId =
-      "b9a727771337fd8ce54821f4805e2d451c4739e92fec6f8ecdb18ff9d1983b27";
+      "eb45b20cc979d504f96b9efc9a08c48103c6f017afa09c0df5c70a5f92a98ea8";
```

Comment:
Mind adding test cases for the new functionality you added?

Reply:
Added.